### ISSUE TITLE
Error while building VAPOR from source on a mac

### ISSUE THREAD

**Issue by NihanthCW** [NCAR/VAPOR#3114]
### Describe the bug
I'm following the instructions found here: https://vapor.readthedocs.io/en/readthedocs/contributing.html 

cmake <source> fails with the following error message related to Qt (fwiw, I am using the libraries supplied by VAPOR as mentioned in the link above)

OS: OSX Big Sur 11.6.5

### To Reproduce
Steps to reproduce the behavior. For example:
1. Follow instructions given here 'https://vapor.readthedocs.io/en/readthedocs/contributing.html ' to build #3109 
4. See error https://github.com/NCAR/VAPOR/pull/3109#issuecomment-1148087316 

### Desktop
 - OS and version: OSX Big Sur 11.6.5
 - Version: #3109

**Comment by sgpearse** [comment]
@sgpearse needs to revisit the "build from source" instructions.

Additionally, the instructions need to be expanded to be a) build gui instructions, b) build python instructions.

**Comment by NihanthCW** [comment]
@sgpearse This is the error.

**Comment by NihanthCW** [comment]
> Hey @NihanthCW, the problem seems to be an OSX "quarantine" flag that gets put on some executable applications. After re-downloading the third party library bundle, I can reproduce your error.
> 
> I've updated the OSX 3rd party libraries after removing this flag on all files. Could you try extracting [this tar.xz](https://drive.google.com/file/d/1Q-IXlP_OgZSXsWKmT-smyrW9xnR-dUfg/view?usp=sharing) file into /usr/local/VAPOR-Deps and run cmake again?
> 
> I'm not sure if OSX puts this flag on executables during file extraction. If that's the case, you'll need to run the following command to remove the flag, and I'll add this step to our documentation. "sudo" is tricky on OSX. I was able to run it from my admin account, but in the past I've needed to take extra steps. Let me know if it doesn't work and I can fill in more details.
> 
> `sudo xattr -dr com.apple.quarantine /usr/local/VAPOR-Deps/2019-Aug`

Perfect! This worked. Thanks Scott! 
- As you've mentioned, it appears OSX is adding the flag during extraction
- I logged in a superuser (su ..) and was able to execute the xattr command

Regarding python api. The build completed successfully with and without the BUILD_PYTHON=ON flag. 
- I got an error about missing "cppyy" when running the example notebooks. I was able to run the examples once I installed cppyy. Is this something that needs to be in the doc? copying @StasJ 

Bottom line: I am able to launch Vapor GUI and run python examples on my Mac now! So, thats great!

**Comment by StasJ** [comment]
> I got an error about missing "cppyy" when running the example notebooks. I was able to run the examples once I installed cppyy. Is this something that needs to be in the doc? copying @StasJ

Currently, the documentation is only for building the Vapor GUI and CPPYY is only required for the python API. We will be updating it to cover building the Python API for the 3.7 release.

**Comment by NihanthCW** [comment]
Sounds good. I'll add this comment to the documentation issue for reference. 
Thanks guys!

**Comment by clyne** [comment]
@sgpearse regarding the use of `xattr -dr com.apple.quarantine`: do we have this in our "build from source" documentation?. If not, can you please open an issue to add?

**Comment by sgpearse** [comment]
@stasj That's correct - the Python api build documentation will be addressed in #3148.

@clyne - I've gone ahead and added a note on it [here](https://vapor.readthedocs.io/en/readthedocs/contributing.html#building-third-party-libraries-not-recommended) in our contributor's guide.

**Comment by clyne** [comment]
Great. thanks.
